src/frozen_solvers.jl

```diff
@@ -171,7 +174,7 @@ The Butcher tableau reads (see tableau (5) of [EngøMarthinsen:1998](@cite)):
 The last row is used for error estimation.
 """
```

Comment:
(since manifolds and retractions show up often, we could even check whether. it makes sense to move lines like  https://github.com/JuliaManifolds/Manopt.jl/blob/39c10e3172612a46b85d599feab2af45b082dcc9/src/documentation_glossary.jl#L381-L393 over to ManifoldsBase for easier / unified docs. Might also be tough since the links are maybe different)

Reply:
There is a lot to document so maybe let's not delay this PR until I manage to do that?

Reply:
Ok. Then let's just keep that in mind to at some point document this more :)